src/modules/Dashboard/features/Applet/Schedule/Schedule.tsx

```diff
@@ -36,16 +36,17 @@ export const Schedule = () => {
   );
 
   useEffect(() => {
-    if (!appletId || !hasAccess) return;
+    if (!appletId || !hasAccess || !respondentId || !ownerId) return;
```

Comment:
pls move ownerId and respondentId check after the dispatch(getEvents({ appletId, respondentId })), respondentId can be undefined inside of getEvents

and why do we need here getRespondentDetails?

Reply:
fixed